backends/tfhe-cuda-backend/cuda/include/integer/integer_utilities.h

```diff
@@ -6019,9 +6017,7 @@ template <typename Torus> struct int_prepare_count_of_consecutive_bits_buffer {
         params.glwe_dimension, params.polynomial_size, params.message_modulus,
         params.carry_modulus, generate_bi_lut_lambda, allocate_gpu_memory);
 
-    if (allocate_gpu_memory) {
```

Comment:
We never have these if conditions in the rest of the code and I think it's fine, because broadcast lut will only copy anything if the memory was allocated.